app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -124,6 +124,7 @@ class ExecApplication extends AbstractApplication<ExecContext> {
 		println """
 Usage:
 $baseCommand
+	 [-args-file <Path>]
```

Comment:
Please correct the indendation here, and also below for the filters. There should only be spaces used, no tabs.